hudi-common/src/test/java/org/apache/hudi/common/util/TestFSUtils.java

```diff
@@ -74,7 +74,7 @@ public void testMaskFileName() {
   }
 
   @Test
-  /**
```

Comment:
Based on other test code style, we can let the `@Test` annotation close to the method so that we can keep `/**` here. WDYT?

Reply:
ok